src/whatsnew/3.0.rst

```diff
@@ -0,0 +1,398 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _release/3.0.x:
+
+============
+3.0.x Branch
+============
+
+.. contents::
+    :depth: 1
+    :local:
+
+.. _release/3.0.x/upgrade:
+
+Upgrade Notes
+=============
+
+.. rst-class:: open
+
+* :ghissue:`2228`: The default maximum document size has been reduced to 8MB. This means
+  that databases with larger documents will not be able to replicate into CouchDB 3.0
+  correctly without modification. This change has been made in preparation for
+  anticipated hard upper limits on document size imposed by CouchDB 4.0. For 3.x,
+  the max document size setting can be relaxed via the ``[couchdb] max_document_size``
+  config setting.
+* :ghissue:`2228`: The default database sharding factor ``q`` has been reduced to 2 by
+  default. This, combined with automated database resharding (see below), is a better
+  starting place for new CouchDB databases. As in CouchDB 2.x, specify ``?q=##`` to
```

Comment:
```suggestion
  starting place for new CouchDB databases. As in CouchDB 2.x, specify ``?q=#`` to
```

maybe just one placeholder to avoid confusion